fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -324,12 +322,10 @@ protected void touchParent( final String id ) {
 
     /* Overriding so unit test can mock. */
     @Override
-    @VisibleForTesting
```

Comment:
I'm actually inclined to close this PR, leaving the @VisibleForTesting annotation in place. The issue I'm trying to resolve at present relates to version mismatches 18.0 != 16.0.1, but @VisibleForTesting is available in both versions, so even if we downgraded to guava 16.0.1, we could still use this annotation.